_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
+| {% include anchor a="error.consumption.attributes.predecessorIsNotThirdPartyOwnedRelationshipAttribute" %}      | The predecessor is not a [third party owned RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                       |
+| {% include anchor a="error.consumption.attributes.predecessorSourceAttributeIsNotRepositoryAttribute" %}        | The predecessor source Attribute is not a [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute).                                                                                                                                                                          |
+| {% include anchor a="error.consumption.attributes.predecessorSourceContentIsNotEqualToCopyContent" %}           | The predecessor source Attribute `content` doesn't match the content of the predecessor shared Attribute copy. ([Update Attributes by succession]({% link _docs_integrate/update-attributes-by-succession.md %}))                                                                                           |
+| {% include anchor a="error.consumption.attributes.senderIsNotPeerOfSharedAttribute" %}                          | The sender of the Attribute is not the peer you shared the Attribute with.                                                                                                                                                                                                                                  |
+| {% include anchor a="error.consumption.attributes.setPredecessorIdDoesNotMatchActualPredecessorId" %}           | The predecessor's `id` and the explicitly set value for the successor's succeeds field don't match.                                                                                                                                                                                                         |
+| {% include anchor a="error.consumption.attributes.successionMustNotChangeContentType" %}                        | The successor Attribute's content type does not match that of the predecessor. An Attribute succession must not change the Attribute's content type, i.e. an IdentityAttribute must not be succeeded by a RelationshipAttribute and v.v.                                                                    |
+| {% include anchor a="error.consumption.attributes.successionMustNotChangeKey" %}                                | The predecessor Attribute's `key` does not match that of the successor. The succession of a RelationshipAttribute must not change the `key`.([Succeeding a RelationshipAttribute]({% link _docs_integrate/update-attributes-by-succession.md %}#succeeding-a-relationshipattribute-and-notifying-the-peer)) |
```

Comment:
As this error is specific to RelationshipAttributes, write "RelationshipAttribute" instead of "Attribute". Also, use a link to the data object RelationshipAttribute.

Missing links also in other descriptions of error messages. Always link to a data object the first time it occurs within a description, but at least if you are referring to one of its properties. For example, the `key` of a [RelationshipAttribute].